README.md

```diff
@@ -46,11 +46,12 @@ they will just retry until it does.
 
 ## Build
 
-Build:
+- `rm Makefile.main; rm -rf .ci` to make sure you will have the last Makefile changes.
+- `make dependencies` to download vendor data using Glide.
+- `make packages` to generate binaries for several platforms.
 
-`make packages`
-
-You will find binaries in `borges_linux_amd64/borges` and `borges_darwin_amd64/borges`.
+You will find binaries in `borges_linux_amd64/borges` and `borges_darwin_amd64/borges`. 
+Into `bin/borges` you will find the binary for the platform used to build the artifact.
```

Comment:
`bin/borges` is always `linux-amd64`. I guess this might change in the future, but maybe you can leave it out of the README?